src/app/containers/NewStoryPromo/index.jsx

```diff
@@ -0,0 +1,38 @@
+// This is the controller; What promo should I use?
+// Should the controller also take care of List or singleItem?
+import React from 'react';
+import { string, bool } from 'prop-types';
+import TopPromo from './PromoTypes/TopPromo';
+import RegularPromo from './PromoTypes/RegularPromo';
+import LeadingPromo from './PromoTypes/LeadingPromo';
+import TopStoriesPromo from './PromoTypes/TopStoriesPromo';
+import RelatedContentPromo from './PromoTypes/RelatedContentPromo';
+
+const promo = {
+  top: TopPromo,
+  regular: RegularPromo,
+  leading: LeadingPromo,
+  topStories: TopStoriesPromo,
+  relatedContent: RelatedContentPromo,
+};
+
+// More props to come ofc :D
+const StoryPromo = ({ promoType, hasImage, hasMediaIndicator }) => {
```

Comment:
I'm not sure I'd necessarily continue to have a storypromo as such. I'd just use build an the relevant promo types close where they are used and co-locate them with the Page they are used on. I like the individual PromoTypes just not sure this layer is needed to bring it together?